src/components/Dropdown/Dropdown.tsx

```diff
@@ -101,49 +100,57 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
     toggleDropdown(false);
   }, { disabled: !isActive });
 
-  useGlobalListener('keydown', e => {
-    if (e.key === 'ArrowDown' || e.key === 'ArrowUp') {
-      e.preventDefault();
-    }
-
-    if (!isActive) {
-      return;
-    }
+  useEffect(() => {
+    const dropdownContainer = containerRef.current;
+    dropdownContainer?.addEventListener('keydown', handleKeyDown);
```

Comment:
Is it possible to use `onKeyDown` on the container div instead of `addEventListener`?

Reply:
yup, updated!